src/components/InputComponent/index.ts

```diff
@@ -0,0 +1,33 @@
+import Block from '../../utils/Block';
+import template from './template';
+import styles from './style.module.less';
+
+interface InputComponentProps {
+    title?: string;
+    name?: string;
+    type?: string;
```

Comment:
Для таких данных, как тип чего-то, ```string``` не очень подходит. Тип - это ограниченное количество значений. ```string``` допускает любые значения. Тут лучше подойдет использование union'a
```suggestion
  type?: "number" | "file";
```

Reply:
Интересная мысль, я об этом не подумал
`type?: "text" | "password" | "email" | "tel" |"number" | "file";`